utils.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/command/rubycommand"
+	"github.com/bitrise-io/go-utils/log"
+	version "github.com/hashicorp/go-version"
+)
+
+// IsToolInstalled ...
+func IsToolInstalled(name, version string) (bool, error) {
+	return rubycommand.IsGemInstalled(name, version)
+}
+
+// IsXcprettyInstalled ...
+func IsXcprettyInstalled() (bool, error) {
+	return IsToolInstalled("xcpretty", "")
+}
+
+// InstallXcpretty ...
+func InstallXcpretty() error {
+	cmds, err := rubycommand.GemInstall("xcpretty", "")
+	if err != nil {
+		return fmt.Errorf("Failed to create command model, error: %s", err)
```

Comment:
do not return start the error message with upper case letter, if outside you log the error with upper cased porefix:
`Failed to install xcpretty, error: Failed to create command model, error: ...`